test/integration/consul-container/test/debugging.md

```diff
@@ -0,0 +1,73 @@
+# Remote Debugging Integration Tests
+
+- [Introduction](#introduction)
+	- [How it works](#how-it-works)
+- [Getting Started](#getting-started)
+	- [Prerequisites](#prerequisites)
+	- [Running Upgrade integration tests](#debugging-integration-tests)
+      -  [Building images](#building-images)
+	  -  [Remote debugging using GoLand](#remote-debugging-using-goland)
+
+
+## Introduction
+
+Remote debugging integration tests allows you to attach your debugger to the consul container and debug go code running on that container. 
+
+### How it works
+The `dev-docker-dbg` Make target will build consul docker container that has the following:
+- [delver (dlv) debugger](https://github.com/go-delve/delve) installed.
```

Comment:
```suggestion
- [delve (dlv) debugger](https://github.com/go-delve/delve) installed.
```

Reply:
Thank you!  Included in 34bd61e44f2b3de4deec9e8d10c9df4ef1e1c528